ghproxy/ghproxy.go

```diff
@@ -90,6 +91,15 @@ func (o *options) validate() error {
 	return nil
 }
 
+func (o *options) Config() *config.Config {
```

Comment:
I don't feel strong on returning a partial Config struct here even though the configGetter interface used by the pusher only uses the PushGateway but I can't think of anything else now. @cjwagner thoughts?